src/components/core/menu/mainMenu/MainMenuLeft.tsx

```diff
@@ -16,23 +16,35 @@ export const MainMenuLeft: FC = () => {
       data-testid="main-nav"
     >
       <Link
-        to={PathNames.strategies}
+        to={'/'}
         onClick={() => carbonEvents.navigation.navHomeClick(undefined)}
       >
         <LogoCarbon className={'w-34'} />
       </Link>
 
       <div className={'hidden space-x-24 md:block'}>
-        {menuItems.map(({ label, href, hrefMatches }, index) => {
-          const isSamePage = isPathnameMatch(pathname, href, hrefMatches);
+        {menuItems.map(({ label, href }, index) => {
+          let isSamePage = match({
+            to: href,
+            search: {},
+            params: {},
+            fuzzy: true,
+          });
+          if (pathname.startsWith('/strategies') && href === '/') {
+            isSamePage = true;
+          }
```

Comment:
Might be more readable to extract think into a dedicated function outside of the template.
```typescript
const isSamePageLink = (to: string) => {
  if (pathname.startsWith('/strategies') && href === '/') return true;
  return match({ to, search: {}, params: {}, fuzzy: true });
} 
...
{menuItems.map(({ label, href }, index) => {
  const isSamePage = isSamePageLink(href);
})
```